backend/src/routes/v0/settings.py

```
@@ -0,0 +1,82 @@
+from src.services.db import get_store
+
+router = APIRouter(tags=["Settings"])
+
+
+def _get_repo(user: User, store) -> UserSettingsRepo:
+    return UserSettingsRepo(str(user.id), store)
+
+
+@router.get("/settings", response_model=UserSettingsResponse)
+async def get_settings(
+    user: User = Depends(verify_credentials), store=Depends(get_store)
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)